### PR TITLE
[FIX] Fix for issue #382

### PR DESCRIPTION
**Pull request by wpichler**

### CODE REVIEW

File: hr_timesheet_sheet_attendance/models/hr_timesheet_sheet.py [hunk 1]
```diff
@@ -84,6 +84,7 @@ def check_employee_attendance_state(self):
     def create(self, vals):
         res = super(HrTimesheetSheet, self).create(vals)
         attendances = self.env['hr.attendance'].search([
+            ('employee_id', '=', vals['employee_id']),
```
Comment (astirpe) [inline hr_timesheet_sheet_attendance/models/hr_timesheet_sheet.py]: You should get the employee_id from res:

```suggestion
            ('employee_id', '=', res.employee_id.id),
```

and I think this is the reason why the tests are failing.

**Review by astirpe — CHANGES_REQUESTED** [review]

File: hr_timesheet_sheet_attendance/models/hr_timesheet_sheet.py (hunk 1 above)
Comment (wpichler) [reply hr_timesheet_sheet_attendance/models/hr_timesheet_sheet.py]: Thanks for this - i've changed this now

**Review by astirpe — APPROVED** [review]

**Review by dreispt — APPROVED** [review]

**Pull request merged**